src/sass/abstractions/icons/_icon-mixin.scss

```diff
@@ -0,0 +1,108 @@
+@import 'init';
+
+@import '../../dist/svg/svg-sprite-mixins';
+@import '../../dist/fonts/svg-font-mixins';
+
+// g-icon()
+//
+// Global mixin for include colored icon.
+// With modernizr check if browser supports svg-fragment and css-mask
```

Comment:
you should add this check then in .modernizrrc

Reply:
its done